tools/utils/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Will it be better to move this package into onnxruntime_extensions folder to be an android sub-package?

Reply:
Those file is copied from `onnxruntime` repo and aim to run Android emulator. Currently, we need to build onnxruntime by ourself, so I leverage files in `onnxruntime` repo and delete those files.